src/main/java/suneido/runtime/builtin/Lucene.java

```diff
@@ -103,8 +109,7 @@ public static Object Insert(Object self, Object key, Object text) {
 			doc.add(new StringField("key", Ops.toStr(key), Field.Store.YES));
 
 			FieldType type = new FieldType();
-	        type.setIndexed(true);
```

Comment:
setIndexed isn't needed any more?

Reply:
It isn't even a method of field type anymore. Unless specified to not have an index, it seems to be indexed